framework/components/ema-separator/ema-separator.twig

```diff
@@ -25,13 +25,9 @@
 {# Print result #}
 <div class="{{ _css_class }}"{{ _extra_attributes }}>
   <div class="ema-separator__heading">
-    <h2 class="ema-separator__heading--title">
+    <h3 class="ema-separator__heading--title ecl-heading ecl-heading--h3">
```

Comment:
Sorry, I forgot that we have custom headings on EMA.
So classes should be ema-heading ema-heading--h3, not the ecl ones.

Once done we can merge it.